packages/base/src/config/CalendarType.ts

```diff
@@ -1,14 +1,14 @@
 import CalendarType from "../types/CalendarType.js";
 import { getCalendarType as getConfiguredCalendarType } from "../InitialConfiguration.js";
 
-let calendarType: CalendarType | undefined;
+let calendarType: `${CalendarType}` | undefined;
 
 /**
  * Returns the configured or default calendar type.
  * @public
  * @returns { CalendarType } the effective calendar type
  */
-const getCalendarType = (): CalendarType => {
+const getCalendarType = () => {
```

Comment:
For me we have to refactor this otherwise it appears as:
![image](https://user-images.githubusercontent.com/31909318/229539281-cdd745dd-24c2-42af-96ab-c9f7be0b0761.png)

Reply:
I think this is part of the main question that we have duplication between Enum values, and String literal values.